src/features/locators/components/LocatorIcon.tsx

```diff
@@ -3,22 +3,23 @@ import { Spin, Tooltip } from "antd";
 import Icon from "@ant-design/icons";
 import CheckedEdited from "../assets/checked-edited.svg";
 import WarningEditedSvg from "../assets/warning-edited.svg";
-import { locatorTaskStatus, VALIDATION_ERROR_TYPE } from "../../../common/constants/constants";
+import { locatorTaskStatus } from "../../../common/constants/constants";
 import { PauseCircle, Trash, WarningCircle } from "phosphor-react";
+import { LocatorValidationErrorType, LocatorValidationWarnings, LocatorValue } from "../types/locator.types";
 
-//remove me
-export const VALIDATION_ERROR_MESSAGES = {
-  [VALIDATION_ERROR_TYPE.DUPLICATED_LOCATOR]: "The locator for this element already exists.", // warn
-  [VALIDATION_ERROR_TYPE.EMPTY_VALUE]: "The locator was not found on the page.",
-  [VALIDATION_ERROR_TYPE.MULTIPLE_ELEMENTS]: " elements were found with this locator.", // warn
-  [VALIDATION_ERROR_TYPE.NEW_ELEMENT]: "The locator leads to a new element on the page after editing.", // success
-  [VALIDATION_ERROR_TYPE.NOT_FOUND]: "The locator was not found on the page.", // warn
-};
+export const isEdited = (locator: LocatorValue) => locator.customXpath;
```

Comment:
It isn't used anywhere. Do we still need it?

Reply:
removed